src/plugins/dashboard/server/saved_objects/migrations_730.test.ts

```diff
@@ -95,7 +89,7 @@ test('dashboard migration 7.3.0 migrates filters to query on search source when
     },
   };
 
-  const doc700: DashboardDoc700To720 = migrations['7.0.0'](doc);
+  const doc700 = migrations['7.0.0'](doc, mockContext);
```

Comment:
This migration is now an explicit `SavedObjectMigrationFn` so the second parameter is now needed in the signature, so I had to adapt the tests.